lib/src/components/NxTextInput/types.ts

```diff
@@ -43,12 +45,15 @@ export type Props = Omit<StateProps, 'trimmedValue'> & HTMLProps & {
   onKeyPress?: ((keyCode: string) => void) | null;
   validatable?: boolean | null;
 
-  // For internal use only, this prop allows additional content to be inserted just before the
+  // For internal use only:
+  // This prop allows additional content to be inserted just before the
```

Comment:
Seems like an extraneous change, was there more to this originally?

Reply:
> Seems like an extraneous change, was there more to this originally?

Oh yes the comment here was removed when merged from your branch.
Fixed here: c6e1f4b31240cf0571402551b86aff7470a0e989